view/js/main.js

```diff
@@ -33,6 +33,50 @@ function _resizeIframe(obj, desth) {
 	setTimeout(_resizeIframe, 100, obj, ch);
 }
 
+function getCookie(name) {
+	var value = "; " + document.cookie;
+	var parts = value.split("; " + name + "=");
+
+	if (parts.length == 2) {
+		return parts.pop().split(";").shift();
+	}
+}
+
+function initWidget(inflated, deflated) {
```

Comment:
You shouldn't need an explicit initialization function for this feature. I'd prefer if you used a post-load catchall routine like this:
```javascript
var widgets = document.getElementByClassName('widget');
// Loop over the widgets to initialize them
```
or with jQuery
```javascript`
$(function() {
    $('.widget').each(function () {
        // Loop over the widgets to initialize them
    });
});
```

Reply:
This is too much of JS for me.